exercises/resistor-color-duo/ResistorColorDuo.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class ResistorColorDuo
+{
+    public static int Value(string[] colors) {
+        throw new NotImplementedException("You need to implement this function.");
+    }
```

Comment:
Tiny formatting suggestion:

```suggestion
    public static int Value(string[] colors)
    {
        throw new NotImplementedException("You need to implement this function.");
    }
```